src/shared/ui/Input/ui/InputIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from "react"
+
+import classNames from "classnames"
+
+interface InputIconProps extends PropsWithChildren {
+  onClick: () => void
+}
+
+export const InputIcon = ({ children, onClick }: InputIconProps) => {
+  return (
+    <button type="button" className={classNames("disable-default-style", "input-icon")} onClick={onClick}>
+      <div>{children}</div>
```

Comment:
Not sure why you need to wrap children in a div

Reply:
Done